data_structures/linked-list/node.py

```diff
@@ -0,0 +1,19 @@
+
+class Node:
+    """Class for a node."""
+    def __init__(self, val, next=None):
+        """Initialize the node instance."""
+        self.val = val
+        self._next = next
+
+    def __str__(self):
+        """Return the value as a string."""
+        return '{val}'.format(val=self.val)
+
+    def get_data(self):
```

Comment:
Due to the structure of python, it is not necessary to create getter methods for your properties. You can just access the properties directly off of `Node` instances.